src/main/kotlin/com/autonomousapps/tasks/classAnalysisTasks.kt

```diff
@@ -99,18 +103,19 @@ abstract class JarAnalysisTask @Inject constructor(
 
   @TaskAction fun action() {
     // Output
-    val reportFile = output.getAndDelete()
-    val reportPrettyFile = outputPretty.getAndDelete()
+    val reportFile = output
```

Comment:
the reason I delete at the very start is in case there's some error that occurs during task execution. I want to start each task action by deleting its prior output (if it exists). I want to avoid the potential for erroneous or partial output.